renderer/viewer/three/waypointSprite.ts

```
@@ -19,6 +19,8 @@ export const WAYPOINT_CONFIG = {
     pixelSize: 50,
     paddingPx: 50,
   },
+  // Default visual scale factor (can be overridden globally or per-waypoint)
+  DEFAULT_VISUAL_SCALE: 1.0,
```

ESLint

[error] Don't use a zero fraction in the number.

(unicorn/no-zero-fractions)